### PR TITLE
"polish code"

### PR DESCRIPTION
**Pull request by dzhwinter**
polish executor python code

### CODE REVIEW

File: python/paddle/fluid/executor.py [hunk 1]
```diff
@@ -180,18 +180,8 @@ def to_name_str(var):
...
-        for each in places:
-            p = core.Place()
-            p.set_place(each)
-            act_places.append(p)
-
-        # TODO(dzhwinter) : only use the first place
-        self.executor = core.Executor(act_places[0])
+    def __init__(self, place):
+        self.executor = core.Executor(place)
         self.places = places
```
Comment (typhoonzero) [inline python/paddle/fluid/executor.py]: `places` is removed?

Comment (dzhwinter) [reply python/paddle/fluid/executor.py]: done.

File: python/paddle/fluid/executor.py [hunk 2]
```diff
@@ -180,60 +179,24 @@ def to_name_str(var):
...
-        self.executor = core.Executor(act_places[0])
-        self.places = places
+    def __init__(self, place):
+        self.place = place
+        p = core.Place()
+        p.set_place(place)
+        self.executor = core.Executor(p)
         self.program_caches = dict()
 
-    def aslodtensor(self, data):
```
Comment (typhoonzero) [inline python/paddle/fluid/executor.py]: Please confirm removing parsing lod is OK?

Comment (dzhwinter) [reply python/paddle/fluid/executor.py]: We plan to use the nested numpy array to represent a LodTensor, but now we use two structure to do the same thing-- an array of data, a list of lod.

**Pull request merged**